backend/src/Http/Controller/AiRequestController.php

```
@@ -0,0 +1,101 @@
+use App\Application\Ai\TechnicalSpecificationUnavailable;
+use App\Domain\Request\RequestNotFound;
+use Yii;
+use yii\web\HttpException;
+use yii\web\NotFoundHttpException;
+use yii\web\UnprocessableEntityHttpException;
+
+final class AiRequestController extends ApiController
+{
+    public function behaviors(): array
+    {
+        $behaviors = parent::behaviors();
```

PHPStan (2.2.7)

[error] Call to an undefined static method App\Http\Controller\ApiController::behaviors().


(staticMethod.notFound)